ustring.h

```diff
@@ -11,15 +36,33 @@
 #include <span>
 #include <locale>
 
+#if IS_STANDARDIZED
+
+#define STD std
 namespace std {
 
+#else
+
+#define STD stdx
+namespace stdx {
+
+using namespace std;
```

Comment:
By importing this `namespace` you can accidentally overload a function which is undefined behaviour or "ill-formed" in almost all cases. Further, one loses the information of what came from the standard and what was required to **supplement** the standard. `type_traits` for example might need specialisation or a function may need an overload. By using calls to different `namespace`s we highlight where this additional entities were needed.